src/libstore/binary-cache-store.cc

```diff
@@ -263,15 +263,15 @@ void BinaryCacheStore::addToStore(const ValidPathInfo & info, Source & narSource
     stats.narInfoWrite++;
 }
 
-bool BinaryCacheStore::isValidPathUncached(const StorePath & storePath)
+bool BinaryCacheStore::isValidPathUncached(const StorePath & storePath, const std::string ca)
```

Comment:
nit for later: but `std::string_view` would be better

Reply:
Yes, `const std::string` isn't very useful. Maybe you intended `const std::string &`.

Reply:
actually i just meant the callee shouldn't modify ca - `const std::string &` would let you modify it. but it's confusing right next to `const StorePath &`.